main.py

```diff
@@ -176,7 +178,7 @@ def __init__(self, window):
     window.pushButton_Cancel.clicked.connect(self.stopVideo)
     window.setWindowTitle("Audio Visualizer")
 
-    self.previewWindow = PreviewWindow(self, r"background.jpg")
+    self.previewWindow = PreviewWindow(self, os.path.join(os.path.dirname(os.path.realpath(__file__)), "background.jpg"))
```

Comment:
This should maybe be stored in a variable so it can easily be used for other imported files, components, etc.